docs/topics/fake-app-spec.rst

```diff
@@ -0,0 +1,34 @@
+.. _fake-app-spec:
+
+===============
+ Fake App Data
+===============
+
+The `generate_apps_from_spec` command-line tool loads a JSON file containing an
+array of fake app objects. The fields that can be specified in these objects
+are:
+
+type
+    One of "hosted", "web", or "privileged", to specify a hosted app,
+    unprivileged packaged app, or privileged packaged app.
```

Comment:
The manifest calls it "web": https://developer.mozilla.org/en-US/Apps/Build/Manifest#type - but I see your point, "web" applies to both hosted and packaged apps.